### DISCUSSION TITLE
Bug: rendered+searchable CSVs no longer scroll horizontally on web at desktop sizes

### DISCUSSION THREAD

**Discussion by benlk** [community/community#67969] [Repositories]
### Select Topic Area

Bug

### Body

This CSV renders correctly on Github.com on desktop, but it doesn't scroll horizontally: https://github.com/benlk/columbus-govdelivery-rss/blob/trunk/feeds.csv

This is a problem, because the CSV has way too many columns to display, even when the browser is over 2000px wide.

Things I have tested:

- viewing the page while logged-out
- toggling the Files sidebar
- searching within the CSV
- making my browser wider or narrower, at different breakpoints
- switching to Chrome, Safari, and Firefox while logged out

The cause appears to be from an inline style for `.iqiUJK`, which is a class on the main `<table>` element:

```html
<table data-hpc="true" data-testid="csv-table" class="Box-sc-g0xbh4-0 iqiUJK" style="padding-top: 0px; height: 10064.9px;">
```

```css
.iqiUJK {
    display: block;
    overflow-x: hidden;
    isolation: isolate;
}
```

That style comes from an injected `<style>` tag on the page:

```
<style data-styled="active" data-styled-version="5.3.6"></style>
```

This tag is added by JS at some point in the page load; it's not present when the page source code is viewed.

**Comment by SumanthSShet** [comment]
The problem with your CSV table is that the CSS class .iqiUJK is setting the overflow-x property to hidden. This is preventing the table from scrolling horizontally.

To fix this, you can either:

Remove the overflow-x: hidden property from the .iqiUJK CSS class.
Add a different CSS class to the <table> element that overrides the overflow-x: hidden property from the .iqiUJK CSS class.
For example, you could add the following CSS to your page:

CSS
.csv-table {
  overflow-x: auto;
}
Use code with caution. [Learn more](https://bard.google.com/faq#coding)
Then, you would need to update the <table> element to use the .csv-table CSS class:

HTML
<table data-hpc="true" data-testid="csv-table" class="Box-sc-g0xbh4-0 iqiUJK csv-table">
Use code with caution. [Learn more](https://bard.google.com/faq#coding)
This will override the overflow-x: hidden property from the .iqiUJK CSS class and allow the table to scroll horizontally.

If you are not able to modify the CSS for the page, you can try using a browser extension to override the CSS. For example, the Stylish extension for Chrome allows you to create custom CSS styles for any website.

I hope this helps!

**Comment by wkumler** [comment]
Can confirm I'm seeing the same behavior even across operating systems and browsers. Upvoted hoping for a fix sometime soon!
